SiriusXM Functionality

jplourde5 opened the pull request:
Moved to beta

jplourde5 commented:
Moved to beta.

jishi commented:
I did some changes, couldn't find the Fuse module, seems to be called fuse.js? I changed to that, anyway, not sure why your require('Fuse') worked. 

Moved out the channel list into a separate file as well. Please verify that it still works if possible.

jplourde5 commented:
Now that you mention it, I didn’t push Fuse into GetHub.   This is the one that I used.  It works very well and appears to be very popular.

https://github.com/krisk/Fuse



[cid:image001.gif@01D1BC12.4B79C600]<http://www.infor.com/>

Jim Plourde | VP Cloud Services
mobile: 864.270.7677 | office: 864.546.5089 | jim.plourde@infor.com<mailto:jim.plourde@infor.com> | http://www.infor.com<http://www.infor.com/>



From: Jimmy Shimizu <notifications@github.com>
Reply-To: jishi/node-sonos-http-api <reply@reply.github.com>
Date: Wednesday, June 1, 2016 at 2:19 PM
To: jishi/node-sonos-http-api <node-sonos-http-api@noreply.github.com>
Cc: James Plourde <Jim.Plourde@infor.com>, State change <state_change@noreply.github.com>
Subject: Re: [jishi/node-sonos-http-api] SiriusXM Functionality (#222)


I did some changes, couldn't find the Fuse module, seems to be called fuse.js? I changed to that, anyway, not sure why your require('Fuse') worked.

Moved out the channel list into a separate file as well. Please verify that it still works if possible.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://github.com/jishi/node-sonos-http-api/pull/222#issuecomment-223080453>, or mute the thread<https://github.com/notifications/unsubscribe/ASedkBml00HaWayKQ-jNk3eE5ggxt2COks5qHc1GgaJpZM4IriC4>.
